drivers/ng_at86rf2xx/ng_at86rf2xx.c

```diff
@@ -146,12 +146,10 @@ void ng_at86rf2xx_reset(ng_at86rf2xx_t *dev)
 #endif
     /* enable safe mode (protect RX FIFO until reading data starts) */
     tmp = NG_AT86RF2XX_TRX_CTRL_2_MASK__RX_SAFE_MODE;
+    ng_at86rf2xx_reg_write(dev, NG_AT86RF2XX_REG__TRX_CTRL_2, tmp);
```

Comment:
if you put it like this, you can omit the tmp variable and join the two lines above...